docs/tools/eosio-sdk-libraries.md

```diff
@@ -4,52 +4,53 @@ title: EOSIO SDK Libraries
 sidebar_label: EOSIO SDK Libraries
 ---
 
-APIs for integrating with EOSIO blockchains
+## SDK Libraries for EOSIO
```

Comment:
repetitive title?

Reply:
SDK and Library comes to be the same thing?

Reply:
I change it to:
1. SDK for EOSIO
1. Libraries for EOSIO
1. SDK and Libraries for EOSIO

Reply:
SDKs for EOSIO